x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvAssert.java

```diff
@@ -79,12 +79,7 @@ private static void assertMetadata(
         var expectedTypes = expected.columnTypes();
 
         assertThat(
-            format(
-                null,
-                "Different number of columns returned; expected [{}] but actual was [{}]",
-                expectedNames.size(),
-                actualNames.size()
-            ),
+            format(null, "Different number of columns returned; expected {} but actual was {}", expectedNames, actualNames),
```

Comment:
I just wanted the names instead of the counts. They fit on the screen so that seemed fine.